commit c064f94fd6e3c9c7ef7d241a5c81d300ee4b1595
Author: Sergei Trofimovich <siarheit@google.com>
Date:   Sat Apr 16 16:13:58 2016 +0100

    Fix doctests failure when testpack or checkers are installed
    
    Test fails as:
      Test suite doctests: RUNNING...
    
      test/Servant/API/ContentTypesSpec.hs:31:18:
        Ambiguous module name `Test.QuickCheck.Instances':
          it was found in multiple packages:
          checkers-0.4.4@check_A5bAKHstANbBRqwFoOaIKx testpack-2.1.3.0@testp_BjTqfpWNTOG5Lwlc3iqqG9 quickcheck-instances-0.3.12@quick_3Tkh09kYN8p78zxMKFPcZI
      Test suite doctests: FAIL
    
    Fixed by importing 'Test.QuickCheck.Instances' from "quickcheck-instances".
    
    Signed-off-by: Sergei Trofimovich <siarheit@google.com>

diff --git a/test/Servant/API/ContentTypesSpec.hs b/test/Servant/API/ContentTypesSpec.hs
index 062b6b2..e29900e 100644
--- a/test/Servant/API/ContentTypesSpec.hs
+++ b/test/Servant/API/ContentTypesSpec.hs
@@ -3,6 +3,7 @@
 {-# LANGUAGE DeriveGeneric         #-}
 {-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE OverloadedStrings     #-}
+{-# LANGUAGE PackageImports        #-}
 {-# LANGUAGE PolyKinds             #-}
 {-# OPTIONS_GHC -fno-warn-orphans #-}
 module Servant.API.ContentTypesSpec where
@@ -28,7 +29,7 @@ import           GHC.Generics
 import           Network.URL               (exportParams, importParams)
 import           Test.Hspec
 import           Test.QuickCheck
-import           Test.QuickCheck.Instances ()
+import "quickcheck-instances" Test.QuickCheck.Instances ()
 
 import           Servant.API.ContentTypes
 
